build/ci/cloudbuild.py.yaml

```diff
@@ -31,5 +31,5 @@ steps:
     args:
       - -c
       - |
-        ./run_test.sh --setup_python
+        ./run_test.sh --setup_website
```

Comment:
can this line be deleted now that ./run_test.sh -p will create the venv if it doesn't exist? are there any benefits to pre-installing it as precursor step?

Reply:
Good idea, removed.